app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, counter_cache: :comments_counter
+end
```

Comment:
- [ ] [REQUIRED] Also, take into consideration that a part of the project requirements states that the Comment model should have a method that updates the comments counter for a post.

### Hint:

```ruby
def update_comments_counter
  Code here ...
end
```

- [ ] [OPTIONAL] As I have recommended above, I would like to again suggest that you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private.

```ruby
after_save :update_comments_counter

  private

  def update_comments_counter
   Code here ...
  end
```